src/com/sun/jna/CallbackReference.java

```diff
@@ -69,6 +67,20 @@
             throw new Error("Error looking up CallbackProxy.callback() method");
         }
     }
+    
+    private static final Class<Callback> DLL_CALLBACK_CLASS;
```

Comment:
@matthiasblaesing What is your tip for Generics? Remove or use a ?, or maybe a ? extends Callback?

Reply:
`private static final Class<?> DLL_CALLBACK_CLASS;`

`Class#forName` can't know what class is loaded, so either use the raw type `Class` or the unbounded wildcard. Given that raw types should be avoided if possible, the unbounded wildcard is the right choice here.

Reply:
Nice! I need be honest with you, I'm always using github editor 😅